spot_hardware_interface/src/spot_hardware_interface.cpp

```diff
@@ -301,11 +288,13 @@ hardware_interface::return_type SpotHardware::write(const rclcpp::Time& /*time*/
   }
 
   for (std::size_t i = 0; i < info_.joints.size(); ++i) {
-    command_states_.position.at(i) = hw_commands_[interfaces_per_joint_ * i];
-    command_states_.velocity.at(i) = hw_commands_[interfaces_per_joint_ * i + 1];
-    command_states_.load.at(i) = hw_commands_[interfaces_per_joint_ * i + 2];
+    joint_commands_.position.at(i) = hw_commands_[command_interfaces_per_joint_ * i];
+    joint_commands_.velocity.at(i) = hw_commands_[command_interfaces_per_joint_ * i + 1];
+    joint_commands_.load.at(i) = hw_commands_[command_interfaces_per_joint_ * i + 2];
+    joint_commands_.k_q_p.at(i) = hw_commands_[command_interfaces_per_joint_ * i + 3];
+    joint_commands_.k_qd_p.at(i) = hw_commands_[command_interfaces_per_joint_ * i + 4];
```

Comment:
With the constants, all the codes will not rely on a specific order of the lines.